server/startup/i18n.js

```diff
@@ -21,9 +22,29 @@ export function loadCoreTranslations() {
         if (~file.indexOf("json")) {
           Logger.debug(`Importing Translations from ${file}`);
           const json = fs.readFileSync(i18nFolder + file, "utf8");
-          Reaction.Import.process(json, ["i18n"], Reaction.Import.translation);
+          const content = JSON.parse(json);
+
+          Assets.update({
+            type: "i18n",
+            name: content[0].i18n
+          }, {
+            $set: {
+              content: json
+            }
+          }, {
+            upsert: true
+          });
         }
       }
+
+      Assets.find({ type: "i18n" }).forEach((t) => {
+        Logger.info(`Importing ${t.name} translations...`);
```

Comment:
Can we update `translations...`  to just be `translation`. Minor detail, but inline with the rest of the startup messaging.